exercises/concept/top-scorers/tests/Tests.elm

```diff
@@ -0,0 +1,82 @@
+module Tests exposing (..)
+
+import Dict
+import Expect
+import Test exposing (..)
+import TopScorers exposing (..)
+import TopScorersSupport exposing (..)
+
+
+tests : Test
+tests =
+    describe "Top Scorers"
+        [ describe
+            "1. Aggregate scorers"
```

Comment:
All the task-level descriptions [should be a single digit](https://github.com/mpizenberg/elm-test-runner/pull/9/files)
```suggestion
            "1"
```
Yet another reminder that we need better docs :)